modules/varnish/data/csp.yaml

```
@@ -69,7 +69,7 @@ img-src:
   - 'data:'
   - '*.miraheze.org'
   - '*.betaheze.org'
-  - 'upload.wikimedia.org'
+  - '*.wikimedia.org'
```

```suggestion
  - 'upload.wikimedia.org'
  - 'wikimedia.org'
```
this just seems to need `'upload.wikimedia.org'` and `'wikimedia.org'` (the root) according to the CSP warnings for the reasoning for this PR.